README.md

```diff
@@ -100,6 +100,14 @@ Yusuke Wada <https://github.com/yusukebe>
 
 _RegExpRouter_, _SmartRouter_, _LinearRouter_, and _PatternRouter_ are created by Taku Amano <https://github.com/usualoma>
 
+## Local Development
```

Comment:
Could you write these lines to [`docs/CONTRIBUTING.md`](https://github.com/honojs/hono/blob/main/docs/CONTRIBUTING.md)? We don't have to edit README for these lines.